internal/formats/spdx22json/model/doc_element_id.go

```diff
@@ -0,0 +1,12 @@
+package model
+
+import "github.com/anchore/syft/internal/formats/common/spdxhelpers"
+
+// DocElementID represents the identifier string portion of an SPDX document
+// identifier. It should be to used to reference any other SPDX document.
+// DocElementIDs should NOT contain the 'DOCUMENTREF-' portion.
+type DocElementID string
+
+func (d DocElementID) String() string {
```

Comment:
👍 Thanks for carving out the doc_element pathway here. I agree with the design choice of this being an external document and not being categorized/merged into the main package list.